app/components/user-navigation/template.hbs

```diff
@@ -0,0 +1,44 @@
+  {{#if isLoggedIn}}
+    <div class="user-nav-logged-in">
+      {{#click-outside action=(action 'closePopup')}}
```

Comment:
couldn't you do this w/o the extra action by using the `action` and `mut` helpers? i.e. `action=(action (mut isPopupOpen) false)`?